### PR TITLE
Changes: grammatical fixes, and mention of fixed bug with the whatsapp voice messages

### PR DESCRIPTION
**Pull request by zstanecic**
### Link to issue number:
none
### Summary of the issue:
This pull request adds mention about the fact that the whatsapp voice messages no longer freeze nvda as per #15169, and #14888.
This pull request also fixes some mistakes and typos
### Description of user facing changes
Fixes for the documentation
### Description of development approach
Edited the english changes.t2t 
### Testing strategy:
tested locally that the HTML is generated 
### Known issues with pull request:
None
### Code Review Checklist:

- [x] Documentation:
  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] API is compatible with existing add-ons.
- [x] Security precautions taken.

### CODE REVIEW

**Comment by zstanecic** [comment]
Hi,
lets comment all your points
• Credits needs to be asked by users themselves. So you should not add credit for other people than yourself, overall without asking them (or did you?).
Why i should ask them? It is in a good faith, moral to mention the credits. I thought that it is unintentional and forgotten.
@leonardder 
has a github name by all lowercase. as the documentation uses github names as a references to contributor, it is healthier to fix this.

**Comment by zstanecic** [comment]
Done, reverted. 

 



-- 
Ta wiadomość e-mail została sprawdzona pod kątem wirusów przez oprogramowanie antywirusowe Avast.
www.avast.com

**Comment by LeonarddeR** [comment]
I have just fixed the casing of my user name, changed it to @LeonarddeR

**Comment by zstanecic** [comment]
Okay. 

Just reverted



-- 
Ta wiadomość e-mail została sprawdzona pod kątem wirusów przez oprogramowanie antywirusowe Avast.
www.avast.com

File: user_docs/en/changes.t2t [hunk 1]
```diff
@@ -65,8 +65,8 @@ It is also updated with UIA enabled, when typing text and braille is tethered to
 - It is not possible anymore to overwrite NVDA's Python console history. (#15792, @CyrilleB79)
 - In Word, the landing cell will now be correctly reported when using the native Word commands for table navigation ``alt+home``, ``alt+end``, ``alt+pageUp`` and ``alt+pageDown``. (#15805, @CyrilleB79)
 - NVDA now resumes audio if the configuration of the output device changes or another application releases exclusive control of the device. (#15758, #15775, @jcsteh)
-- NVDA should remain responsive when being flooded with many UI Automation events, e.g. large amounts of text in a terminal. (#14888)
- - This new behavior can be disabled using the new Use enhanced event processing setting in NVDA's advanced settings.
+- NVDA should remain responsive when being flooded with many UI Automation events, e.g. when the large pieces of text are sent to terminal or when listening to voice messages in WhatsApp messenger. (#14888, #15169)
+  - This new behavior can be disabled using the new Use enhanced event processing setting in NVDA's advanced settings.
```
Comment (seanbudd) [inline user_docs/en/changes.t2t:69]: the list terminator below must match the indentation change

File: user_docs/en/changes.t2t [hunk 2]
```diff
@@ -65,8 +65,8 @@ It is also updated with UIA enabled, when typing text and braille is tethered to
 - It is not possible anymore to overwrite NVDA's Python console history. (#15792, @CyrilleB79)
 - In Word, the landing cell will now be correctly reported when using the native Word commands for table navigation ``alt+home``, ``alt+end``, ``alt+pageUp`` and ``alt+pageDown``. (#15805, @CyrilleB79)
 - NVDA now resumes audio if the configuration of the output device changes or another application releases exclusive control of the device. (#15758, #15775, @jcsteh)
-- NVDA should remain responsive when being flooded with many UI Automation events, e.g. large amounts of text in a terminal. (#14888)
- - This new behavior can be disabled using the new Use enhanced event processing setting in NVDA's advanced settings.
+- NVDA should remain responsive when being flooded with many UI Automation events, e.g. when the large pieces of text are sent to terminal or when listening to voice messages in WhatsApp messenger. (#14888, #15169)
```
Comment (seanbudd) [inline user_docs/en/changes.t2t:68]: Grammar fixes:

```suggestion
- NVDA should remain responsive when being flooded with many UI Automation events, e.g. when large chunks of text are printed to a terminal or when listening to voice messages in WhatsApp messenger. (#14888, #15169)
```

File: user_docs/en/changes.t2t (hunk 1 above)
Comment (seanbudd) [inline user_docs/en/changes.t2t:69]:
```suggestion
  - This new behavior can be disabled using the new "Use enhanced event processing" setting in NVDA's advanced settings.
```

**Comment by zstanecic** [comment]
it should be applied now along with resolved conflicts from master

**Comment by zstanecic** [comment]
@seanbudd 
This is now ready for potential review or merging. All suggestions applied.

**Pull request merged**